src/duckling/game/collision/collision-types.service.ts

```diff
@@ -46,7 +46,7 @@ export class CollisionTypesService {
         }
         let unknownCollisionTypes = this._findUnknownCollisionTypes(map);
         this._registerMultipleCollisionTypes(unknownCollisionTypes);
-        this._notifyUnknownCollisionTypes(unknownCollisionTypes);
+        this._notifyUnknownCollisionTypes(Array.from(unknownCollisionTypes.values()));
```

Comment:
You can simplify some code if you restore change unknownCollisionTypes back to returning an array.